scripts/m4t/audio_to_units/README.md

```diff
@@ -0,0 +1,19 @@
+# Convert raw audio into units (unit_extraction)
+
+Raw audio needs to be converted to units to train UnitY models and vocoders. Units act as supervision for UnitY models, and are the input to the vocoders which synthesize speech from these units.
+
+The unit extraction pipeline comprises the following steps:
+- Compute features from layer 35 (determined empirically) of the pretrained XLSR v2 model, which is a wav2vec2 model at the core.
+- Assign features for each timestep to a collection of precomputed K-Means centroids to produce a sequence of units.
```

Comment:
Nit: Mention the practice is similar to extracting Hubert units in https://arxiv.org/pdf/2107.05604.pdf